glean-core/src/lib_unit_tests.rs

```diff
@@ -1203,3 +1213,48 @@ fn internal_pings_can_be_disabled() {
     let submitted = glean.internal_pings.baseline.submit_sync(&glean, None);
     assert!(!submitted);
 }
+
+#[test]
+fn pings_are_controllable_from_remote_settings_config() {
+    let _ = env_logger::builder().is_test(true).try_init();
+
+    let dir = tempfile::tempdir().unwrap();
+    let (mut glean, _t) = new_glean(Some(dir));
+
+    let disabled_ping = PingType::new("custom-disabled", true, false, true, true, vec![], false);
+    glean.register_ping_type(&disabled_ping);
+    let enabled_ping = PingType::new("custom-enabled", true, false, true, true, vec![], true);
+    glean.register_ping_type(&enabled_ping);
+
+    // We need to store a metric as an empty ping is not stored.
```

Comment:
Even if you change the second boolean to `true` in the ctor?

Reply:
I think I was intending to extend this to ensure that the metric was cleared when `submit()` was called on the disabled ping, but I think I forgot to come back and do that. Let me do that now, lest I forget again.

Reply:
Or remove it and test that in a separate test